students/cfarris/lessons/lesson03/assignment/test_basic_operations.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env Python3
+import pytest
+from basic_operations import add_customer, search_customer, delete_customer
+from customer_model import Customer
+
+##note, if you get an integrity error be sure clear your database. 
```

Comment:
Would, in fact, be good to have some test fixtures that create a database at the beginning and tear it down at the end. :)